network/containerizer/bridgepolicy.go

```diff
@@ -80,15 +96,22 @@ func (p *BridgePolicy) FindMissingBridgesForContainer(
 			}
 		}
 	}
-	notFound := containerSpaces.Difference(spacesFound)
-	fanNotFound := containerSpaces.Difference(fanSpacesFound)
+
+	// TODO (manadart 2019-09-27): This is ugly, but once everything is
+	// consistently reasoning about spaces in terms of IDs, we should implement
+	// this kind of diffing on SpaceInfos.
+	// network.QuoteSpaceSet can be removed at that time too.
+	guestSpaceSet := set.NewStrings(guestSpaces.Names()...)
+	notFound := guestSpaceSet.Difference(spacesFound)
+	fanNotFound := guestSpaceSet.Difference(fanSpacesFound)
+
 	if p.containerNetworkingMethod == "fan" {
 		if fanNotFound.IsEmpty() {
 			// Nothing to do, just return success
 			return nil, 0, nil
 		} else {
 			return nil, 0, errors.Errorf("host machine %q has no available FAN devices in space(s) %s",
-				m.Id(), network.QuoteSpaceSet(fanNotFound))
+				host.Id(), network.QuoteSpaceSet(fanNotFound))
```

Comment:
Drive-by fix opportunity: the else block is redundant and can be replaced with the return statement. Also, you can convert the following nested else block into an `else if` block

Reply:
Done.